requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
-#    pip-compile --generate-hashes requirements.in
+#    pip-compile --allow-unsafe --generate-hashes requirements.in
```

Comment:
Unsure if the changes here are intentional.

Reply:
Yes they are mandatory with python 3.12 (from #54)